crates/control/src/referee_pose_detection_filter.rs

```diff
@@ -82,37 +93,40 @@ impl RefereePoseDetectionFilter {
             .player_referee_detection_times
             .fill_if_subscribed(|| self.detection_times);
 
+        context
+            .referee_pose_queue
+            .fill_if_subscribed(|| self.own_detected_above_arm_poses_queue.clone());
+
         Ok(MainOutputs {
             is_referee_initial_pose_detected: is_referee_initial_pose_detected.into(),
+            is_own_referee_initial_pose_detected: is_own_referee_initial_pose_detected.into(),
         })
     }
 
-    fn update(&mut self, context: &CycleContext<impl NetworkInterface>) -> Result<()> {
-        let time_tagged_persistent_messages =
-            unpack_message_tree(&context.network_message.persistent);
+    fn update(&mut self, context: &CycleContext<impl NetworkInterface>) -> Result<bool> {
```

Comment:
You don't need the `Result`, just return `bool`

Reply:
Done